app/main.py

```diff
@@ -1,3 +1,27 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        user_and_shop_data = json.load(file)
+    fuel_price = user_and_shop_data["FUEL_PRICE"]
+    customers = [Customer.make_instance(customer) for customer
+                 in user_and_shop_data["customers"]]
+    shops = [Shop.make_instance(shop) for shop in user_and_shop_data["shops"]]
+    is_first_customer = True
+    for customer in customers:
+        if not is_first_customer:
+            print("")
```

Comment:
Sorry, I don’t get this `is_first_customer` Could you explain plese?

Reply:
I use this variable because there should be a blank line between the message about each user. If I print an empty line at the end of a message, then at the very end of the message I will have one extra empty line. so I check if the user is the first and in this case I don't print an empty line before him, but before all the rest I print. this way I avoid the extra line at the very end